diff --git a/third_party/dom_distiller_js/protoc_plugins/json_values_converter.bat b/third_party/dom_distiller_js/protoc_plugins/json_values_converter.bat
--- a/third_party/dom_distiller_js/protoc_plugins/json_values_converter.bat
+++ b/third_party/dom_distiller_js/protoc_plugins/json_values_converter.bat
@@ -1,3 +1,3 @@
 @echo off
 
-call python3 %~dp0\json_values_converter.py
+python3.exe "%~dp0json_values_converter.py" %*
diff --git a/tools/protoc_wrapper/protoc-gen-ts_proto.bat b/tools/protoc_wrapper/protoc-gen-ts_proto.bat
--- a/tools/protoc_wrapper/protoc-gen-ts_proto.bat
+++ b/tools/protoc_wrapper/protoc-gen-ts_proto.bat
@@ -4,4 +4,4 @@
 :: Use of this source code is governed by a BSD-style license that can be
 :: found in the LICENSE file.
 
-python3 "%~dp0protoc-gen-ts_proto.py" %*
\ No newline at end of file
+"%~dp0..\..\third_party\node\win\node.exe" "%~dp0..\..\third_party\node\node_modules\ts-proto\protoc-gen-ts_proto" %*
